extensions/wikia/NjordPrototype/NjordController.class.php

```diff
@@ -7,6 +7,22 @@ class NjordController extends WikiaController {
 
 	const MAINPAGE_PAGE = 'mainpage';
 
+	private static $wikiDataModel = null;
+
+	public function __construct() {
+		parent::__construct();
+
+		self::$wikiDataModel = $this->getWikiDataModel();
```

Comment:
Is it possible to use a non-static var? This assignment should be appropriate for init() method. You might also want to check how many instances of this controller are dispatched